lib/src/ui/pages/dashboard/news/latest/latest_news_page.dart

```diff
@@ -37,16 +38,21 @@ class _LatestNewsListPageState extends State<LatestNewsListPage> {
   }
 
   Widget _buildContent(BuildContext context) {
-    final state = context.select((LatestNewsNotifier notifier) => notifier.state);
-    if (state is NotifierLoadingViewState || state is NotifierInitialViewState) {
-      return const Center(child: CircularProgressIndicator());
-    } else if (state is NotifierErrorViewState) {
-      return _buildErrorContent(context);
-    } else if (state is NotifierLoadedViewState) {
-      return _buildLoadedContent((state as NotifierLoadedViewState<NewsPaginatedViewEntities>).data);
-    } else {
-      return Container();
-    }
+    return ChangeNotifierProvider.value(
```

Comment:
jeżeli tu providujesz notifiera to już nie trzeba tego robić przy nawigacji do tego widoku